plugins/templates/updateTrusted.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const baseSchema = schema.models.template.base;
+
+module.exports = () => ({
+    method: 'PUT',
+    path: '/templates/{name}/trusted',
+    config: {
+        description: "Update a template's trusted property",
+        notes: 'Returns null if successful',
+        tags: ['api', 'templates', 'trusted'],
+        auth: {
+            strategies: ['token'],
+            scope: ['admin', '!guest']
+        },
+        plugins: {
+            'hapi-swagger': {
+                security: [{ token: [] }]
+            }
+        },
+        handler: async (request, reply) => {
+            const { name } = request.params;
+            const { templateFactory } = request.server.app;
+            const { trusted } = request.payload;
+
+            // get the earliest entry
+            const templates = await templateFactory.list({
```

Comment:
not sure if this is any better (probably not): https://github.com/screwdriver-cd/models/blob/master/lib/templateFactory.js#L222

Reply:
i think the sole purpose here is to get the earliest one, dont really need a flexibility like that function there right?